src/main/java/RestApi/movies/controller/MovieController.java

```diff
@@ -1,37 +1,39 @@
 package RestApi.movies.controller;
 
-
-import RestApi.movies.model.Movies;
-import com.fasterxml.jackson.databind.ObjectMapper;
+import RestApi.movies.model.Movie;
+import RestApi.movies.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
+
 import org.springframework.web.bind.annotation.RestController;
 
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
+
 
 @RestController
 public class MovieController {
 
+    @Autowired
+    private MovieService movieService;
+
     @GetMapping
     @RequestMapping("/movies")
-    public void getAll() throws IOException {
-
-        byte[] jsonData = Files.readAllBytes(Paths.get("movies.json"));
-
-        ObjectMapper objectMapper = new ObjectMapper();
-
-        Movies movies = objectMapper.readValue(jsonData, Movies.class);
+    public Movie[] getAll() throws IOException {
 
-        System.out.println(movies);
+            Movie[] response = movieService.findAll().getMovies();
+            return response;
```

Comment:
Should we be returning a `ResponseEntity` here?

Reply:
Yes i'll change it whilst im doing the exception handling